service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseInitiationControllerTest.java

```diff
@@ -415,30 +311,18 @@ private void verifyTaskListUpdated(CaseDetails caseDetails) {
             anyMap());
     }
 
-    private void givenPRDWillFail() {
-        Request request = Request.create(GET, "", Map.of(), new byte[]{}, UTF_8, null);
-        givenPRDWillAnswer(invocation -> {
-            throw new FeignException.NotFound("", request, new byte[]{});
-        });
-    }
-
     private void givenPRDWillReturn(List<String> userIds) {
         givenPRDWillAnswer(invocation -> organisation(userIds));
     }
 
-    private void givenPRDWillAnswer(Answer... answers) {
+    private void givenPRDWillAnswer(Answer<?>... answers) {
```

Comment:
this should be not used now. It is used in single place, which can be written much more cleaner. Was like this to record sequence of responses, now as there is not retries it is not needed

Reply:
as discussed , removed givenPRDWillReturn and replaced givenPRDWillReturn  with your suggestion : 
private void givenPRDWillReturn(List<String> userIds) {
    given(organisationApi.findUsersInOrganisation(USER_AUTH_TOKEN, SERVICE_AUTH_TOKEN, Status.ACTIVE, false)).willReturn(organisation(userIds));
}